tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2015",
```

Comment:
we need this to iterate sets, but this might decrease our browser coverage. can we maybe refactor to use Set.values() while iterating instead?